### ISSUE TITLE
allow_unreachable=False)  # allow_unreachable flag RuntimeError: CUDNN_STATUS_INTERNAL_ERROR

### ISSUE THREAD

**Issue by codermckee** [wutianyiRosun/PSPNet_PyTorch#3]
I have this problem. and My environment is Pytorch 0.4.0.
`File "/home/mckee/Semantic_Segmentation/PSPNet_PyTorch/train.py", line 349, in main
    loss.backward()
  File "/usr/local/lib/python2.7/dist-packages/torch/tensor.py", line 93, in backward
    torch.autograd.backward(self, gradient, retain_graph, create_graph)
  File "/usr/local/lib/python2.7/dist-packages/torch/autograd/__init__.py", line 89, in backward
    allow_unreachable=False)  # allow_unreachable flag
RuntimeError: CUDNN_STATUS_INTERNAL_ERROR`

**Comment by chenyzh28** [comment]
Hi, have you successfully reproduced the results in the paper?

**Comment by codermckee** [comment]
@chenyzh28  No, I didn't reproduce the results. I just use this network to train my own project.

**Comment by Water2style** [comment]
> @chenyzh28 No, I didn't reproduce the results. I just use this network to train my own project.

老哥成功了吗

**Comment by zhuruzhao** [comment]
@Water2style 老哥有没有windows下pytorch能跑的pspnet啊？

**Comment by pioneer158** [comment]
我在Win10上可以训练，但测试的时候，这一行代码：model.load_state_dict(convert_state_dict(saved_state_dict["model"]) )
报错，求解答！
